apiserver/src/main/java/org/dependencytrack/resources/v1/MetricsResource.java

```diff
@@ -140,17 +147,16 @@ public Response getPortfolioMetricsSince(
         if (since == null) {
             return Response.status(Response.Status.BAD_REQUEST).entity("The specified date format is incorrect.").build();
         }
-        try (QueryManager qm = new QueryManager()) {
-            final List<PortfolioMetrics> metrics = qm.getPortfolioMetricsSince(since);
-            return Response.ok(metrics).build();
-        }
+        List<PortfolioMetrics> metrics = withJdbiHandle(handle ->
+                handle.attach(MetricsDao.class).getPortfolioMetricsSince(since.toInstant()));
+        return Response.ok(metrics).build();
     }
 
     @GET
     @Path("/portfolio/{days}/days")
     @Produces(MediaType.APPLICATION_JSON)
     @Operation(
-            summary = "Returns X days of historical metrics for the entire portfolio",
+            summary = "Returns X (30 by default) days of historical metrics for the entire portfolio",
```

Comment:
To my understanding the 30 day default no longer applies since that's controlled by the frontend now, is that correct?

Reply:
Correct, I'll remove it 👍🏼